x/accesscontrol/types/message_dependency_mapping.go

```diff
@@ -131,5 +133,51 @@ func ValidateWasmDependencyMapping(mapping acltypes.WasmDependencyMapping) error
 	if lastAccessOp.Operation.AccessType != acltypes.AccessType_COMMIT {
```

Comment:
we can also verify there is no duplicate message name like we do with normal access ops